features/glance.feature

```diff
@@ -0,0 +1,39 @@
+@glance
+Feature: Glance
+  As an administrator
+  I want to make sure that Glance component is working correctly
+
+  @create_kvm_image
```

Comment:
OK thanks for the info.  I don't know enough about `cct` yet to offer anything concrete yet, but I suspect there may be a way to do this in a more general fashion which scales easily by not requiring manual tagging of each scenario.  For example, http://stackoverflow.com/questions/825748/how-do-i-pass-command-line-arguments-to-a-rake-task